pkg/resource/helper.go

```diff
@@ -68,11 +69,12 @@ func (r *helper) cacheOpenAPISchema() error {
 }
 
 // NewHelperFromRESTConfig returns a new object that allows apply and patch operations
-func NewHelperFromRESTConfig(restConfig *rest.Config, logger log.FieldLogger) (Helper, error) {
+func NewHelperFromRESTConfig(restConfig *rest.Config, fake bool, logger log.FieldLogger) (Helper, error) {
```

Comment:
imo, we should create a separate function `NewFakeHelperFromRESTConfig` . fake argument in new helper seems out of place, a separate function will make the intent and calling easier to understand.

Reply:
That does sound better.